app/controllers/time_summary_controller.rb

```diff
@@ -201,29 +133,26 @@ def time_summary_csv
 
     def time_summary_header_csv
       [
+        t('time_summary.downloads.csv.user'),
         t('time_summary.downloads.csv.date'),
         t('time_summary.downloads.csv.task'),
         t('time_summary.downloads.csv.quantity_hours_per_day')
       ]
     end
 
     def time_summary_data_csv
-      row = []
-
-      (@start_date..@end_date).each do |date|
-        if date.workday?
-          if @items[date].present?
-            @items[date].each do |item, hours|
-              row << [
-                date,
-                item.to_s,
-                helpers.number_with_precision(hours, precision: 1)
-              ]
-            end
-          else
-            row << [date, '', 0]
+      row  = []
+
+      @self_and_descendants.each do |u|
+        TimeConsumption.where(user: u).
+          where(date: @start_date..@end_date).each do |tc|
+            row << [
+              tc.user.full_name,
```

Comment:
```suggestion
              user.full_name,
```
Sino lo busca en cada vuelta (porque aparte no hay un includes).